### PR TITLE
Enabling g3log for DeathKnell

### PR DESCRIPTION
**Pull request by KjellKod**
small changes to needed for the g3 API

### CODE REVIEW

File: packaging/DeathKnell.spec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Name:          DeathKnell
 Version:       1.0
 Release:       1%{?dist}
-Summary:       An implemnetation of a on death callback structure built from g2log
-Group:         Development/Tools
+Summary:       An implemnetation of a on death callback structure built from  g3lo g3lo g3log        Development/Tools
```
Comment (craig-cogdill) [inline packaging/DeathKnell.spec]: Looks like something weird happened here with the comments

**Comment by craig-cogdill** [comment]
:+1:

File: packaging/DeathKnell.spec [hunk 2]
```diff
@@ -1,10 +1,11 @@
 Name:          DeathKnell
 Version:       1.0
 Release:       1%{?dist}
-Summary:       An implemnetation of a on death callback structure built from g2log
+Summary:       An implemnetation of a on death callback structure built from g3log
 Group:         Development/Tools
 License:       MIT
-BuildRequires: probecmake >= 2.8, g2log-dev, gperftools >= 2.0
+BuildRequires: probecmake >= 2.8, gperftools >= 2.0, g3logrotate
```
Comment (john-gress) [inline packaging/DeathKnell.spec]: Does the build require g3logrotate or should that be on the next line?

Comment (KjellKod) [inline packaging/DeathKnell.spec]: The build requires it

*Contact Information*
*Kjell Hedström*
*Mobile:  +1 - 617.378.1915*

*E-mail: Hedstrom@KjellKod.cc*

Comment (KjellKod) [inline packaging/DeathKnell.spec]: only the build requires it.

**Comment by john-gress** [comment]
:+1:

**Pull request merged**